addons/website_event/controllers/main.py

```diff
@@ -1,25 +1,19 @@
 # -*- coding: utf-8 -*-
 
-import babel.dates
-import time
-import werkzeug.urls
+import werkzeug
 from datetime import datetime, timedelta
```

Comment:
@WHPODOO Can I have more explanation about how can we add days using Odoo **fields**